packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsFiltersSection.tsx

```diff
@@ -0,0 +1,132 @@
+          >
+            Choose how posts are ordered in your feed.
+          </Typography>
+        </div>
+        <Dropdown
+          className={{
+            container: 'w-full tablet:max-w-70',
+          }}
+          selectedIndex={Math.max(
+            Object.values(FeedOrder).indexOf(data.orderBy),
+            0,
+          )}
+          options={Object.keys(FeedOrder)}
+          onChange={(_, index) => {
+            const newOrderBy = Object.values(FeedOrder)[index];
+
+            setData({
+              orderBy: newOrderBy === FeedOrder.Recommended ? null : newOrderBy,
+            });
+          }}
+        />
+      </div>
+      <div className="flex flex-col gap-4">
+        <div className="flex flex-col gap-1">
+          <Typography bold type={TypographyType.Body}>
+            Filter by time range
+          </Typography>
+          <Typography
+            type={TypographyType.Callout}
+            color={TypographyColor.Tertiary}
+          >
+            Include only posts published within a specific time range.
+          </Typography>
+        </div>
+        <Radio
+          name="theme"
+          options={feedRangeFilters}
+          value={data.maxDayRange?.toString() || null}
+          onChange={(value) => {
+            setData({ maxDayRange: +value || null });
+          }}
+        />
+      </div>
+      <div className="flex flex-col gap-4">
+        <div className="flex flex-col gap-1">
+          <Typography bold type={TypographyType.Body}>
+            Set content thresholds
+          </Typography>
+          <Typography
+            type={TypographyType.Callout}
+            color={TypographyColor.Tertiary}
+          >
+            Define the minimum upvotes or views posts need to appear in your
+            feed.
+          </Typography>
+        </div>
+        <div className="flex gap-4">
+          <TextField
+            className={{
+              container: 'w-full tablet:max-w-70',
+            }}
+            leftIcon={<UpvoteIcon />}
+            defaultValue={feed.flags?.minUpvotes}
+            name="name"
+            type="number"
+            inputId="feedMinUpvotes"
+            label="Min upvotes"
+            placeholder="No limit"
+            required
+            min={0}
+            max={1000}
+            valueChanged={(value) => setData({ minUpvotes: +value })}
+          />
+          <TextField
+            className={{
+              container: 'w-full tablet:max-w-70',
+            }}
+            leftIcon={<EyeIcon />}
+            defaultValue={feed.flags?.minViews}
+            name="name"
+            type="number"
+            inputId="feedMinViews"
+            label="Min views"
+            placeholder="No limit"
+            required
+            min={0}
+            max={1000}
+            valueChanged={(value) => setData({ minViews: +value })}
+          />
+        </div>
+      </div>
+      <div className="flex flex-col gap-4">
```

Comment:
I think we shouldn't need the gap here as it is the only child. Maybe even the wrapper 👀

Reply:
yes!